### ISSUE TITLE
State "absent" not supported by fmgr_pm_pkg_scopemember

### ISSUE THREAD

**Issue by cr0cdev** [fortinet-ansible-dev/ansible-galaxy-fortimanager-collection#82]
Hi!

I've found out that `fmgr_pm_pkg_scopemember` does not support `state: absent` parameter even though the documentation says it is supported.
This is what I get when I try to remove a ADOM from global policy after unassigning the policy from ADOM.
Task:
```
- name: Remove ADOM from global policy
  fortinet.fortimanager.fmgr_pm_pkg_scopemember:
    workspace_locking_adom: "global"
    adom: "global"
    pkg_path: "{{ global_policy }}"
    state: absent
    pm_pkg_scopemember:
      name: "{{ adom_name | lower }}"
``` 
Response:
```
fatal: [fortimanager01]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "access_token": null,
            "adom": "global",
            "bypass_validation": false,
            "enable_log": false,
            "forticloud_access_token": null,
            "pkg_path": "GLOBAL-POLICY",
            "pm_pkg_scopemember": {
                "name": "ansible-test",
                "vdom": null
            },
            "proposed_method": null,
            "rc_failed": null,
            "rc_succeeded": null,
            "state": "absent",
            "workspace_locking_adom": "global",
            "workspace_locking_timeout": 300
        }
    },
    "msg": "This module doesn't not support state:absent yet."
}

``` 

Not sure if `fmgr_pm_pkg_scopemember` only supports ADOM level policy members (_devices_) or it also supports removing ADOM from global policy.

Thanks!

**Comment by dux-fortinet** [comment]
Hi cr0cdev,

Thanks for pointing it out. Some of the modules indeed have the "state" parameter while don't support absent (delete).
I just checked `fmgr_pm_pkg_scopemember` manually. This module doesn't support `state:absent` in the current FortiManager Ansible Collection version.
I will improve our code and this module will be able to use `state:absent` in the next version.

Thanks

**Comment by dux-fortinet** [comment]
This problem has been fix in FortiManager Ansible Collection 2.5.0
You can use `state:absent` for module `fmgr_pm_pkg_scopemember` now. :)

**Comment by cr0cdev** [comment]
Thanks Dux! Closing this now :)
